### PR TITLE
Add flake8 linter ignore support

### PR DESCRIPTION
**Pull request by RFRIEDM-Trimble**
This solves #423

### CODE REVIEW

**Review by mjeronimo — APPROVED** [review]

**Comment by RFRIEDM-Trimble** [comment]
My bad. Missed this when I commented out some code right before push. Will fix. 
`2: cmake/ament_cmake_flake8_lint_hook.cmake:18: Line ends in whitespace [whitespace/eol]`

**Comment by RFRIEDM-Trimble** [comment]
Fixed the white space, tests should pass.

File: ament_cmake_flake8/cmake/ament_flake8.cmake [hunk 1]
```diff
@@ -57,7 +59,9 @@ function(ament_flake8)
   if(ARG_MAX_LINE_LENGTH)
     list(APPEND cmd "--linelength" "${ARG_MAX_LINE_LENGTH}")
   endif()
-  list(APPEND cmd ${ARG_UNPARSED_ARGUMENTS})
+  if(ARG_EXCLUDE)
+    list(APPEND cmd "--exclude" "${ARG_EXCLUDE}")
+  endif()
```
Comment (aprotyas) [inline ament_cmake_flake8/cmake/ament_flake8.cmake:64]: You still want to add `ARG_UNPARSED_ARGUMENTS` to your command once you're done processing all arguments.


```suggestion
  if(ARG_EXCLUDE)
    list(APPEND cmd "--exclude" "${ARG_EXCLUDE}")
  endif()
  
  list(APPEND cmd ${ARG_UNPARSED_ARGUMENTS})
```

**Review by aprotyas — CHANGES_REQUESTED** [review]
Small modification requested by otherwise LGTM

**Comment by aprotyas** [comment]
Can someone run CI?

**Review by aprotyas — APPROVED** [review]
@RFRIEDM-Trimble approving preemptively, mind fixing the EOL whitespace issue and amending the commit message to satisfy DCO?

**Comment by RFRIEDM-Trimble** [comment]
> @RFRIEDM-Trimble approving preemptively, mind fixing the EOL whitespace issue and amending the commit message to satisfy DCO?

Done, the whitespace was already fixed btw.  I didn't realize accepting change suggestions in the UI that it doesn't sign it for me. Sorry for troubles.

**Comment by mjeronimo** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/17789/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/12335/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/18439/)

**Pull request merged**